reana_workflow_controller/app.py

```diff
@@ -42,6 +42,34 @@
 }
 
 
+@app.route('/workflows', methods=['GET'])
+def get_workflows():
+    """Get all workflows.
+    
+    .. http:get:: /workflows
+        Returns a JSON list with all the workflows.
+        NOTE: Currently just a mock that can used e.g. to quickly
```

Comment:
Regarding to the NOTE, it is rendered all in the same line and it doesn't look very pretty. I would do either one of the following things:
- Add empty line between the endpoint description and the note.
```
        Returns a JSON list with all the workflows.

        NOTE: Currently just a mock that can used e.g. to quickly
        check if reana-workflow-controller is running.
```
- Use `sphinx` `.. warning::` so it gets nicely rendered with red background.
```
        .. warning::
            Not implemented, just a mock that can be used e.g. to quickly
            check if reana-workflow-controller is running.
```
p.s. mind the missing verb in `... just a mock that can [be] used ...`